src/main/java/frc/trigon/robot/commands/commandfactories/GeneralCommands.java

```diff
@@ -1,13 +1,13 @@
-package frc.trigon.robot.commands;
+package frc.trigon.robot.commands.commandfactories;
 
 import edu.wpi.first.wpilibj2.command.*;
 import frc.trigon.robot.RobotContainer;
-import frc.trigon.robot.constants.CommandConstants;
+import frc.trigon.robot.commands.CommandConstants;
 import frc.trigon.robot.subsystems.MotorSubsystem;
 
 import java.util.function.BooleanSupplier;
 
-public class Commands {
+public class GeneralCommands {
```

Comment:
add a jdoc to explain the difference from the command constants, and refer to the command constants doc here.